### PR TITLE
feat(storage): fill cache with priority

### PR DESCRIPTION
**Pull request by Little-Wallace**
I hereby agree to the terms of the [RisingWave Labs, Inc. Contributor License Agreement](https://gist.github.com/TennyZhuang/f00be7f16996ea48effb049aa7be4d66#file-rw_cla).

## What's changed and what's your intention?

In our system the data in l0 would be access  more probably,  so we can fill them with higher priority

## Checklist For Contributors

- [x] I have written necessary rustdoc comments
- [x] I have added necessary unit tests and integration tests
- [x] I have added fuzzing tests or opened an issue to track them. (Optional, recommended for new SQL features #7934).
- [x] I have demonstrated that backward compatibility is not broken by breaking changes and created issues to track deprecated features to be removed in the future. (Please refer to the issue)
- [x] All checks passed in `./risedev check` (or alias, `./risedev c`)

## Checklist For Reviewers

- [x] I have requested macro/micro-benchmarks as this PR can affect performance substantially, and the results are shown.

## Documentation

- [x] My PR **DOES NOT** contain user-facing changes.

<details><summary>Click here for Documentation</summary>

### Types of user-facing changes

Please keep the types that apply to your changes, and remove the others.

- Installation and deployment
- Connector (sources & sinks)
- SQL commands, functions, and operators
- RisingWave cluster configuration changes
- Other (please specify in the release note below)

### Release note

</details>

### CODE REVIEW

File: src/common/src/cache.rs [hunk 1]
```diff
@@ -142,6 +145,30 @@ impl<K: LruKey, T: LruValue> LruHandle<K, T> {
         }
     }
 
+    fn is_high_priority(&self) -> bool {
+        (self.flags & IS_HIGH_PRI) > 0
+    }
+
+    fn set_high_priority(&mut self, high_priority: bool) {
```
Comment (Li0k) [inline src/common/src/cache.rs:152]:
```rust
set_priority(CachePriority)
```

File: src/storage/src/hummock/sstable_store.rs [hunk 2]
```diff
@@ -105,7 +105,8 @@ pub enum CachePolicy {
     /// Disable read cache and not fill the cache afterwards.
     Disable,
     /// Try reading the cache and fill the cache afterwards.
-    Fill,
+    ///  true means it will fill a high priority block.
+    Fill(bool),
```
Comment (chenzl25) [inline src/storage/src/hummock/sstable_store.rs:109]: Due to the ambiguity between Fill(false) and NotFill, how about using a new enum `Priority(High, Low)` to replace `Fill(bool)` with `Fill(Priority)`?

**Review by chenzl25 — APPROVED** [review]
LGTM!

**Review by Li0k — APPROVED** [review]
Rest LGTM

File: src/storage/src/hummock/sstable_store.rs (hunk 2 above)
Comment (Little-Wallace) [reply src/storage/src/hummock/sstable_store.rs]: Good idea.

**Pull request merged**